### PR TITLE
fcs_gate_trans tool

### PR DESCRIPTION
**Pull request by emily101-gif**

### CODE REVIEW

File: flowtools/fcs_gate_trans/FCSGateTrans.xml
Comment (pcm32) [inline flowtools/fcs_gate_trans/FCSGateTrans.xml:4]: flowcore is a dependency of flowdensity, so we could remove the flowcore dependency. However what I wonder @emily101-gif is if binaries produced by one version of flowcore work well with another version of flowcore. Would you have access to someone who is experienced with these packages to know that? I would expect it to be alright, as FCS seems to be a wider and versioned format, but might be worth asking

**Review by pcm32 — COMMENTED** [review]
Happy to merge once we have an answer to my question on the version issue @emily101-gif.

File: flowtools/fcs_gate_trans/FCSGateTrans.xml
Comment (emily101-gif) [reply flowtools/fcs_gate_trans/FCSGateTrans.xml:4]: Hi Pablo. I would imagine contacting the bioconductor re the flowCore package version would be the best way to go about this. But for some of the tools I have been testing it has been failing when I remove flowCore as a dependency.

Comment (emily101-gif) [reply flowtools/fcs_gate_trans/FCSGateTrans.xml:4]: @pcm32 was just testing - and when I removed the flowCore package as a dependency, the tool ran but no output produced. So perhaps as you mentioned a specific version of flowCore has to be referenced.

Comment (pcm32) [reply flowtools/fcs_gate_trans/FCSGateTrans.xml:4]: Weird, I did remove flowcore and both tests passed, it installed flowcore 1.44 instead of 1.42 on its own.

Comment (pcm32) [reply flowtools/fcs_gate_trans/FCSGateTrans.xml:4]: I have asked here: https://github.com/RGLab/flowCore/issues/193

Comment (pcm32) [reply flowtools/fcs_gate_trans/FCSGateTrans.xml:4]: Based on answer from tool maintainers, will go for just pinning flowdensity. Will merge if tests pass.

**Review by pcm32 — APPROVED** [review]

**Pull request merged**
